### PR TITLE
proj: Bahmni

### PR DESCRIPTION
**Pull request by OscarAblinger**

### CODE REVIEW

File: src/bahmni.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+# Description taken throuhg a synthesis of the github 
+# description and their FAQ
+description: >
+  Bahmi is an Open Source hospital Management System focusing
```
Comment (AdrienLemaire) [inline src/bahmni.yaml]: Bahmni*

File: src/bahmni.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+# Description taken throuhg a synthesis of the github
```
Comment (AdrienLemaire) [inline src/bahmni.yaml]: through*

File: src/bahmni.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  - python
+  - shell
+  - groovy
+# See guideline on how to rate a project
+rating: 5
+repoUrl: "https://github.com/Bahmni"
+sdgs: [3]
+websiteUrl: "https://www.bahmni.org/"
+
+contributionGuidelinesUrl: "https://bahmni.atlassian.net/wiki/spaces/BAH/pages/116069555/Contributing+to+Bahmni"
```
Comment (AdrienLemaire) [inline src/bahmni.yaml:30]: Seeing a white page when trying to directly access this URL. Looks like some JIRA issue. Can we find another link ?

File: src/bahmni.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  It's aimed to being a generic system which can be used for
+  multiple diseases and hospitals in different countries.
+license: AGPL-3.0
+name: Bahmni
+programmingLanguages:
+  - java
+  - javascript
+  - python
+  - shell
+  - groovy
```
Comment (AdrienLemaire) [inline src/bahmni.yaml:17]: Note that these are only the Top languages listed in github. I can see other projects with TSQL, Ruby, Perl and SQLPL.
That's also why I thought of adding the "main: true" optional argument. Some projects are mainly 1 language (eg Java), with a bunch of other languages for minor scripting.

**Review by AdrienLemaire — CHANGES_REQUESTED** [review]
@AblingerOscar do you mind installing `yamllint` and running it (you can make a local git hook if you think you'll forget about it until we add the github Check)?
Also, you might want to include an orthography corrector in your text editor (I personally using vim with the wordy plugin, and the Grammarly chrome extension).

File: src/bahmni.yaml (hunk 3 above)
Comment (OscarAblinger) [reply src/bahmni.yaml:30]: works for me 🤔 
Do you mind checking again to see whether it's a permanent issue or if they were just down for a bit?
Their entire documentation is on the same site, though.
Alternate link would be: https://bahmni.atlassian.net/wiki/spaces/BAH/pages/1310748/Developer+s+Guide but that one is less friendly to read

Not sure what else we could use…

File: src/bahmni.yaml (hunk 4 above)
Comment (OscarAblinger) [reply src/bahmni.yaml:17]: If you want we can re-introduce `main`. I was only against having only one language with it, not multiple.
Would you mind adding an exact definition somewhere of what warrants a `main` tag to you? Just so we can stay consistent :)

**Comment by OscarAblinger** [comment]
Sure, I can do that. Though it would be interesting what configuration you use with it. The default?
I just always ran it through www.yamllint.com, so that seems to have a less strict interpretation.

File: src/bahmni.yaml (hunk 3 above)
Comment (AdrienLemaire) [reply src/bahmni.yaml:30]: I can open and read it from my phone. Maybe a temporary issue indeed. Thanks

**Review by AdrienLemaire — APPROVED** [review]
Sorry, forgot to approve this PR. @AblingerOscar it would help if you could re-request a review from me, by reselecting me as a reviewer. This way, I'll get a notification, and I know you're done with the new modifications :)

**Comment by OscarAblinger** [comment]
Oh sorry, I did not notice that you were removed as an reviewer ^^'

**Comment by AdrienLemaire** [comment]
@AblingerOscar I wasn't removed, but re-assigning me send me a direct notification that I need to do something :)

**Pull request merged**